cmd/juno/juno.go

```diff
@@ -170,6 +172,7 @@ const (
 		"The upper limit is 4 million steps, and any higher value will still be capped at 4 million."
 	corsEnableUsage             = "Enable CORS on RPC endpoints"
 	versionedConstantsFileUsage = "Use custom versioned constants from provided file"
+	pluginPathUsage             = "Path to the plugins .so file"
```

Comment:
do you think `Path to the plugin .so file` would be more correct? I would thought that it's possible to put several plugins in the same .so file

Reply:
I guess you can have multiple plugins (/go types that satisfy the JunoPlugin interface), but Juno is only going to load one of them. Specifically the one whose variable name is "JunoPluginInstance"

Reply:
IIRC, this approach simplified loading and using the plugin, but if you think it's necessary to load multiple plugins, then I should update the load function

Reply:
>but if you think it's necessary to load multiple plugins

I meant maybe we should change comment to "Path to the plugin~s~ .so file" ?